package.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<?xml-model
+  href="http://download.ros.org/schema/package_format3.xsd"
+  schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ed_python</name>
+  <version>0.0.1</version>
+  <description>Python interface to ed_msgs</description>
+
+  <author email="MatthijsBurgh@outlook.com">Matthijs van der Burgh</author>
+
+  <maintainer email="MatthijsBurgh@outlook.com">Matthijs van der Burgh</maintainer>
+
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <depend>ed_msgs</depend>
+
+  <exec_depend condition="$ROS_PYTHON_VERSION == 2">python-numpy</exec_depend>
+  <exec_depend condition="$ROS_PYTHON_VERSION == 3">python3-numpy</exec_depend>
+  <exec_depend condition="$ROS_PYTHON_VERSION == 2">python-yaml</exec_depend>
+  <exec_depend condition="$ROS_PYTHON_VERSION == 3">python3-yaml</exec_depend>
+  <exec_depend>pykdl_ros</exec_depend>
+  <exec_depend>python_orocos_kdl</exec_depend>
+  <exec_depend>rospy</exec_depend>
+  <exec_depend>std_msgs</exec_depend> <!--[TEMP] Waiting for change in ed_msgs -->
```

Comment:
Please make the relevant changes to remove this temporary dependency

Reply:
Changing this will create too much works on other repo's. So to limit this, we should leave this for now.